Zend/zend_compile.h

```diff
@@ -326,6 +326,9 @@ typedef struct _zend_oparray_context {
 /* function is a destructor                               |     |     |     */
 #define ZEND_ACC_DTOR                    (1 << 29) /*     |  X  |     |     */
 /*                                                        |     |     |     */
+/* function is a destructor                               |     |     |     */
```

Comment:
Comment is identical as the define above

Reply:
I'm not sure, if this change is safe.
"prototype" is also used to check overridden protected method visibility.
If you change "prototype" meaning, this also changes visibility.